apps/server/src/modules/tool/controller/tool.controller.spec.ts

```diff
@@ -67,4 +78,33 @@
 			expect(lti11Uc.getLaunchParameters).toHaveBeenCalledWith(currentUser, toolId, courseId);
 		});
 	});
+
+	describe('createExternalTool', () => {
+		const bodyConfig: ExternalToolConfigCreateParams = {
```

Comment:
why do you not use the constructor here and throw away the external-tool-request.spec.ts?